Makefile

```diff
@@ -89,6 +96,13 @@ uninstall:
 	$(call printlog, UNINSTALL, $(INSTALL_BIN)/$(APPS3))
 	$(QUIET) rm -rf $(INSTALL_BIN)/$(APPS3)
 
+build:
+	./kmesh_compile.sh
+	
+docker:
+	make build
+	docker build --build-arg dir=$(DIR) -f kmesh.dockerfile -t $(IMAGE) .
```

Comment:
how can we do multi platform support

Reply:
This function will be supported in the future. The current pr is not affected.

Reply:
> how can we do multi platform support

This function will be supported in the future. The current pr is not affected.